src/main/cpp/subsystems/ScoringSubsystem.cpp

```diff
@@ -95,9 +103,13 @@ void ScoringSubsystem::SpinAmp(double upperPercentage, double lowerPercentage) {
   frc::SmartDashboard::PutNumber("Amp lower target %", lowerPercentage);
 }
 
-void ScoringSubsystem::SpinSpeaker(double upperPercentage, double lowerPercentage) {
-  m_speakerLowerSpinnyBoi.Set(-lowerPercentage);
-  m_speakerUpperSpinnyBoi.Set(-upperPercentage);
+void ScoringSubsystem::SpinSpeaker(double upperPercentage,
+                                   double lowerPercentage) {
+  // m_speakerLowerSpinnyBoi.Set(-lowerPercentage);
```

Comment:
Should this be a macro/flag to use either `Set` or `RunWithVelocity`? Would be the same across all `Spin...` methods.